app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command) > 0:
```

Comment:
You should validate then command length is equal exactly to 3 parts, because you're performing unpacking with 3 elements